start/software/manage-python-packages.md

```diff
@@ -0,0 +1,136 @@
+[title]: - "Run Python Scripts on OSG"
+
+# Run Python Scripts on OSG
+
+[TOC]
+
+## Overview
+
+If your jobs need specific Python packages, this guide will show you how to 
+install these packages to your home directory and use them to run jobs.  
+
+Before we begin, you should know which Python packages you need to run your job.  
+
+## Install and prepare needed Python packages
+
+Load Python.  
+
+     $ module load python/3.7.0
+
+Now, we will go through the steps to create a virtual environment.  The first 
+command creates the base environment. You 
+can swap out `my_env` for a more descriptive name like `scipy` or `word-analysis`.
+
+     $ python3 -m venv testenv
```

Comment:
This should probably say `python3 -m venv my_env` to match with the rest of the document (or switch all references of `my_env` to say `testenv`)

Reply:
oops, you're right.